nx/include/switch/services/nfc.h

```diff
@@ -128,9 +121,7 @@ typedef enum {
     NfpuMountTarget_All = 3,
 } NfpuMountTarget;
 
-const NfpuInitConfig *nfpuGetDefaultInitConfig(void);
-
-Result nfpuInitialize(void);
+Result nfpuInitialize(const void *config, size_t config_size);
```

Comment:
I meant you to remove `nfpuGetDefaultInitConfig`; not to remove the default config itself and add a parameter to nfpuInitialize.